doozerlib/distgit.py

```diff
@@ -981,11 +981,15 @@ def _detect_permanent_build_failures(self, scanner):
     def push(self):
         with Dir(self.distgit_dir):
             self.logger.info("Pushing repository")
-            exectools.cmd_assert("timeout 1200 rhpkg push", retries=3)
-            # rhpkg will create but not push tags :(
-            # Not asserting this exec since this is non-fatal if a tag already exists,
-            # and tags in dist-git can't be --force overwritten
-            exectools.cmd_gather(['timeout', '60', 'git', 'push', '--tags'])
+            try:
+                exectools.cmd_assert("timeout 1200 rhpkg push", retries=3)
+                # rhpkg will create but not push tags :(
+                # Not asserting this exec since this is non-fatal if a tag already exists,
+                # and tags in dist-git can't be --force overwritten
+                exectools.cmd_gather(['timeout', '60', 'git', 'push', '--tags'])
+            except IOError as e:
+                return (self.metadata, repr(e.message))
```

Comment:
`repr`? is `e.message` ever not a string?

Reply:
oops... that was a typo, should've removed `.message`